core/modules/user/user.admin.inc

```diff
@@ -399,14 +399,55 @@ function user_login_settings($form, &$form_state) {
     ),
   );
 
+  $form['user_login_page_settings'] = array(
+    '#type' => 'fieldset',
+    '#title' => t('Login page settings'),
+  );
+  $user_login_page_tab_options = array(
+    'tabs' => t('Use tabs'),
+    'links' => t('Use links'),
+    'simplified' => t('Show simplified page'),
+  );
+  $user_login_page_default_settings = $core_config->get('user_login_page_settings');
+  $form['user_login_page_settings']['user_login_page_settings'] = array(
+    '#title' => t('Login page options'),
+    '#type' => 'radios',
+    '#options' => $user_login_page_tab_options,
+    '#default_value' => $user_login_page_default_settings ? $user_login_page_default_settings : 'tabs',
+    '#config' => 'system.core',
+    'tabs' => array(
+      '#description' => t('Show local tabs for "Create new account", "Log in", and "Reset password" links.'),
+    ),
+    'links' => array(
+      '#description' => t('Show plain links to these pages.'),
+    ),
+    'simplified' => array(
+      '#description' => t('Show plain links to these pages, and hide all layout regions, such as headers, footers, and sidebars.'),
+    ),
+  );
   $form['flood_log_failed_attempts'] = array(
     '#type' => 'checkbox',
     '#title' => t('Log excessive failed login attempts.'),
     '#default_value' => $flood_config->get('flood_log_failed_attempts'),
     '#description' => t('If there are excessive failed login attempts, the offending IP address or user account will be temporarily blocked. By default these events will be logged, as it helps identify brute force login attacks. You may want to disable logging them if for example, you are using the dblog module, and want to avoid database writes.'),
   );
 
-  return system_settings_form($form);
+  $form['actions']['#type'] = 'actions';
+  $form['actions']['submit'] = array(
+    '#type' => 'submit',
+    '#value' => t('Save configuration')
+  );
+
+  return $form;
+}
+
+/**
+ * Form submission handler for user_login_settings().
+ */
+function user_login_settings_submit($form, &$form_state) {
+  system_settings_form_submit($form, $form_state);
+
+  backdrop_flush_all_caches();
```

Comment:
This _should_ rebuild the menu, but it doesn't seem to be working. Would this help?

```suggestion
  backdrop_flush_all_caches();
  state_set('menu_rebuild_needed', TRUE);
```

Reply:
The second line alone does the trick, thx.